src/main/java/de/komoot/photon/query/PhotonQueryBuilder.java

```diff
@@ -84,12 +87,13 @@ private PhotonQueryBuilder(String query, String language) {
 
         m_finalQueryWithoutTagFilterBuilder = new FunctionScoreQueryBuilder(m_query4QueryBuilder, m_alFilterFunction4QueryBuilder.toArray(new FilterFunctionBuilder[0]))
                 .boostMode(CombineFunction.MULTIPLY).scoreMode(ScoreMode.MULTIPLY);
-
+        
         // @formatter:off
         m_queryBuilderForTopLevelFilter = QueryBuilders.boolQuery()
                 .should(QueryBuilders.boolQuery().mustNot(QueryBuilders.existsQuery("housenumber")))
                 .should(QueryBuilders.matchQuery("housenumber", query).analyzer("standard"))
-                .should(QueryBuilders.existsQuery(String.format("name.%s.raw", language)));
```

Comment:
Removing this would seem to partially break querying for objects with names.

Reply:
Thank you! I will do more tests as well.